calendar_event_report/tests/test_weekly_report_wizard.py

```diff
@@ -0,0 +1,146 @@
+# © 2019 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
+
+from odoo.tests import common
+import datetime
+
+
+class TestWeeklyReportWizard(common.TransactionCase):
+
+    def setUp(self):
+        super(TestWeeklyReportWizard, self).setUp()
+        self.Rooms = self.env['resource.calendar.room']
+        self.WeeklyReportWizard = self.env['weekly.report.wizard']
+        self.EMPLOYEE_BASE = self.env.ref(
+            'base.group_user')
+        self.EMPLOYEE_GROUP = self.env.ref(
+            'project_resource_calendar.group_resource_calendar_user')
+        self.RESOURCE_CALENDAR_MANAGER = self.env.ref(
+            'project_resource_calendar.group_resource_calendar_manager')
+        self.RESOURCE_CALENDAR_EDITOR = self.env.ref(
+            'project_resource_calendar.group_resource_calendar_editor')
+        self.room_1 = self.Rooms.create({
+            'name': 'Test Room 1',
+            'resource_type': 'room',
+            'allow_double_book': True,
+        })
+        self.weekly_report_wizard_1 = self.WeeklyReportWizard.create({
+            'room_id': self.room_1.id,
+            'recurrency': True,
+            'state': 'open',
+        })
+        self.Users = self.env['res.users']
+        self.Employees = self.env['hr.employee']
+        self.user_base = self.Users.create({
+            'name': 'Base User',
+            'login': 'base@test.com',
+            'groups_id': [(6, 0, [self.EMPLOYEE_BASE.id])]
+        })
+        self.user_guest = self.Users.create({
```

Comment:
remove users with a reference to project_resource_calendar, move tests with these references to project_specific